.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:9.11.1
+    environment:
+      REPO: "dddecaf/decaf-metabolica"
```

Comment:
The repo should be `gcr.io/dd-decaf-cfbf6/decaf-metabolica` as we are moving our images over to GCR where we have unlimited private repos.